qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -306,12 +311,12 @@
                     gagent_check_type = file_read
                 - with_fsfreeze:
                     gagent_check_type = with_fsfreeze
-                - with_selinux:
```

Comment:
@6-dehan Do you intent to remove `with_selinux`?

Reply:
Uh, I remembered, if we don't test it, we can just keep them here but remove it from our test loop.

Reply:
@6-dehan You could remove it here if the test does not make sense now. I just want to confirm that you do want to remove it rather than a typo or mistake. Thanks.